src/server/debugcmd.cc

```diff
@@ -500,7 +495,19 @@ void DebugCmd::Reload(CmdArgList args) {
   }
 
   string last_save_file = sf_.GetLastSaveInfo().file_name;
-  Load(last_save_file);
+
+  sf_.FlushAll(cntx_);
+
+  if (auto fut_ec = sf_.Load(last_save_file, ServerFamily::LoadExistingKeys::kFail); fut_ec) {
+    GenericError ec = fut_ec->Get();
+    if (ec) {
+      string msg = ec.Format();
+      LOG(WARNING) << "Could not load file " << msg;
```

Comment:
I think it's error

Reply:
I moved this code from below
We shouldn't get alerts for failing debug commands IMO